tests/periph_i2c/README.md

```diff
@@ -1,17 +1,50 @@
 # Periph I2C Test
 
 This application enables you to test all available low-level I2C functions
-using the RIOT shell.  Consult the 'help' shell command for available actions.
+using the RIOT shell. The `robot-test` feature requires a target board to be
+connected to the hardware reference board PHiLIP.
 
-## Background
+## Automated Test Setup
 
-Test for the low-level I2C driver.  Can be run manually or using automated test
-script for RobotFramework, run `make robot-test`.
+Before executing the automated test, the DUT (device under test) and reference
+hardware PHiLIP must be setup.
 
-## Glossary
+### Robot Framework Setup
 
-DEV - The I2C device number, this is usually 0.  Consult the board header file for more information.</br>
-ADDR - The address of the slave I2C device, this is usually the sensor.  Information on the slave address should be found in the sensor datasheet.</br>
-REG - The register of the slave device/sensor.  Not all sensors follow this access method.  A I2C_NOSTOP (0x04) may be needed.</br>
-LEN - The length to read or write.</br>
-FLAG - Flags set for the I2C, more information is available in driver/include/periph/i2c.h
+Follow the [setup instructions](../../README.md) and install all python
+packages needed.
+
+### PHiLIP Setup
+
+To setup PHiLIP follow instructions on the
+[PHiLIP repo](https://github.com/riot-appstore/PHiLIP#setup).
+
+### DUT and PHiLIP Wiring Setup
+
+Each test will need to be wired in a specific way. To wire the DUT follow
+the [guide to board pinouts](http://doc.riot-os.org/group__boards.html).
+To wire PHiLIP check the [pinout](https://github.com/riot-appstore/PHiLIP#pinb)
+for the PHiLIP version used.
+
+This test will require that both `sda` and `scl` pins of the `i2c.dev 0` to
+be wired to the `DUT_SDA` and `DUT_SCL` pins on PHiLIP. If the device has a
+reset pin then the optional `DUT_RST` pin on PHiLIP can be connected.
+
+DUT pin name | PHiLIP pin name | Required
+-------------|-----------------|---------
+sda          | DUT_SDA         | Yes
+scl          | DUT_SCL         | Yes
```

Comment:
shouldn't they also share at least one `GND`?

Reply:
Not really, they don't need to if you are using non-isolated USB.  It would be safer to say that...